packages/calculator/src/__tests__/draggable-calculator.test.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import Draggable from 'react-draggable';
+import TestUtils from 'react-dom/test-utils';
+import ReactDOM from 'react-dom';
+import DraggableCalculator from '../draggable-calculator';
+import assert from 'power-assert';
+
+describe('Render Calculator Component Using Shallow Render', () => {
+  let drag;
+  let wrapper;
+  let x = 0;
+  let y = 0;
+  const props = {
+    mode: 'basic',
+    show: 'true',
+    classes: { handle: '', typography: '' },
+    closeIcon: jest.fn()
+  };
+
+  beforeEach(() => {
+    wrapper = shallow(<DraggableCalculator {...props} />)
+  })
+
+  function mouseMove(x, y, node) {
+    const doc = node ? node.ownerDocument : document;
+    const evt = doc.createEvent('MouseEvents');
+    evt.initMouseEvent('mousemove', true, true, window,
+        0, 0, 0, x, y, false, false, false, false, 0, null);
+    doc.dispatchEvent(evt);
+    return evt;
+  }
+
+  function handleDrag(e,ui){
+    x = x + ui.deltaX;
+    y = y + ui.deltaY;
+    return false;
+  }
+
+  it('Checks For Rendering', () => {
+    expect(wrapper.length).toEqual(1)
+  });
+
+  it('Creates a snapshot for rendering', () => {
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('should immediately call onStop if onDrag returns false', function () {
```

Comment:
Why not just use enzyme.shallow to just test the body of the `handleDrag` function?
```jsx
// 
describe('handleDrag', () => {
  it('updates state.deltaPosition', () => {
     const wrapper = shallow(<DraggableCalculator/>);
     wrapper.instance().handleDrag(null, {deltaX: 10, deltaY: 10});
     const deltaPosition = wrapper.state('deltaPosition');
     expect(deltaPosition).toEqual({x: 10, y: 10});
  })
});
```